launcher/modplatform/atlauncher/ATLPackManifest.cpp

```diff
@@ -178,13 +197,21 @@ static void loadVersionMod(ATLauncher::VersionMod & p, QJsonObject & obj) {
             p.depends.append(Json::requireString(depends));
         }
     }
+    p.colour = Json::ensureString(obj, QString("colour"), "");
+    p.warning = Json::ensureString(obj, QString("warning"), "");
 
     p.client = Json::ensureBoolean(obj, QString("client"), false);
 
     // computed
     p.effectively_hidden = p.hidden || p.library;
 }
 
+static void loadVersionMessages(ATLauncher::VersionMessages& m, QJsonObject& obj)
+{
+    m.install = Json::ensureString(obj, "install", "");
+    m.update = Json::ensureString(obj, "update", "");
+}
+
 void ATLauncher::loadVersion(PackVersion & v, QJsonObject & obj)
```

Comment:
Maybe you could do a comment up here with the URL where this is specified, so that it's easier to reference it later on? (I found it in [here](https://github.com/ATLauncher/ATLauncher/blob/master/src/main/java/com/atlauncher/data/json/Version.java), the wiki seems to be outdated / incomplete? Or I just missed it :P)

Reply:
The JSON models aren't really documented anywhere, but they're pretty close to the XML models - documented https://wiki.atlauncher.com/pack-admin/xml-guide/getting-started.

Reply:
I see, I was looking for the API docs so that I could tell whether it was possible for the response to randomly change "colour" to "color" at some point, breaking stuff with it :smile: 
But if there's no documentation, we'll just have to roll with it i guess :)

Reply:
I can't see Ryan misspelling it after a decade (ATL must be coming up on it if it isn't :p).